docs/src/test-api/class-testconfig.md

```diff
@@ -482,6 +482,27 @@ export default defineConfig({
 ```
 
 
+## property: TestConfig.shardingMode
+
+* since: v1.46
+- type: ?<[ShardingMode]<"partition"|"round-robin"|"duration-round-robin">>
+
+Defines the algorithm to be used for sharding. Defaults to `'partition'`.
+* `'partition'` - divide the set of test groups by number of shards. e.g. first
+  half goes to shard 1/2 and seconds half to shard 2/2.
+* `'round-robin'` - spread test groups to shards in a round-robin way. e.g. loop
+  over test groups and always assign to the shard that has the lowest number of
+  tests.
+* `'duration-round-robin'` - use duration info from `.last-run.json` to spread
+  test groups to shards in a round-robin way. e.g. loop over test groups and
+  always assign to the shard that has the lowest duration of tests. new tests
+  which were not present in the last run will use an average duration time. When
+  no `.last-run.json` could be found the behavior is identical to
```

Comment:
I disagree.

Also please note that `duration-round-robin` and 'round-robin' are using the same implementation. The only difference is that in case of 'duration-round-robin' the last run info is passed to the implementation. However, in case the last run info is passed but it is empty, the behaviour is identical to 'round-robin' where no last run info is passed.

I don't think it would be a good to choose a different algorithm based on whether last run info is available or not.